frontend/src/App.js

```diff
@@ -25,6 +26,7 @@ class App extends Component {
               )}
             </div>
           </header>
+          <TextField id="outlined-basic" label="Outlined" variant="outlined" />
```

Comment:
here's where I added a text field input to test it out https://mui.com/material-ui/react-text-field/